src/table/pagedTable/pagedTable.tsx

```diff
@@ -66,17 +71,19 @@ export function PagedTable<T>({
     columnsSpan = 1
   }
 
+  const listIsNotEmpty = React.useMemo(() => rowsToDisplay && rowsToDisplay.length, [rowsToDisplay])
+
+  //render manually passed component if the list is empty
+  const renderEmptyComponent = React.useCallback(
+    () => (emptyRowComponent ? emptyRowComponent : <EmptyTableRow colSpan={columnsSpan} rowText={emptyRowText} />),
```

Comment:
should we yell at them if they pass emptyRowComponent and emptyRowText? you can't have both right?

Reply:
right now empty component will just override the text,
how can we tell user that they cant use both?

Reply:
could throw an error